src/modules/announcements/announcement.controller.ts

```diff
@@ -43,13 +50,20 @@ export class AnnouncementController {
   // Fetch a particular Announcement using ID
   @Get('/:announcementId')
   @ApiParam(announcementId)
+  @ApiOperation({
+    summary: 'Fetch a particular Announcement using ID',
+  })
   @ApiOkResponse(responsedoc.getAnnouncement)
   async getAnnouncement(@Param('announcementId') announcementId: string) {
     return await this.announcementService.getAnnouncement(announcementId);
   }
 
+  // update Announcement by Id
   @Put('/:announcementId')
   @ApiParam(announcementId)
+  @ApiOperation({
+    summary: 'update Announcement by Id',
```

Comment:
Since we are adding sumary now, We don't need `//` comments over the top of each controller.
Please remove those, We will just try to give the best summary under text.

Reply:
Done